from finance_data.errors.configuration_error import ConfigurationError
